core/vm/evm.go

```diff
@@ -91,7 +91,7 @@ func run(evm *EVM, contract *Contract, input []byte, readOnly bool) ([]byte, err
 		if evm.chainRules.IsYoloV1 {
 			precompiles = PrecompiledContractsYoloV1
 		}
-		if p := precompiles[*contract.CodeAddr]; p != nil {
+		if p := precompiles[address]; p != nil {
```

Comment:
Why did this change?

Reply:
if you check the latest version, you see that there is not change with Quorum `master`